recipes/skaware/files/HOST_CPU_x86_64/sysdeps.h

```diff
@@ -0,0 +1,49 @@
+/* ISC license. */
```

Comment:
Unfortunately, this is something we need to provide ourselves to add a new arch.  I will add a comment about that.